src/main/java/org/networkcalculus/dnc/network/server_graph/ServerGraph.java

```diff
@@ -741,6 +741,14 @@ public Flow getFlow(int id) throws Exception {
 		}
 	}
 
+	/**
+	 * Finds all the flows that cross at least one server on the path p.
+	 * @param p
+	 *            The path which the returned flows intersect.
+	 * @return Set of flows that cross at least one server on the path.
+	 * @throws Exception
+	 *			   Could not sort although there may be flows.
```

Comment:
Not sure where this description comes from. I checked all other occurrences, it does not seem helpful anywhere. In particular as there are not particular exceptions caught -- at the moment, the DNC's way to handle exceptions is to just hand them to the calling method until we reach the code that started the analysis, so the user can decide what to do with them.
Long story short, I suggest to remove this line and all other occurrences of "Could not sort although there may be flows."